components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.common/src/main/java/org/wso2/carbon/identity/api/server/application/management/common/ApplicationManagementConstants.java

```diff
@@ -70,6 +70,8 @@ private ApplicationManagementConstants() {
     public static final String FAPI_ALLOWED_CLIENT_AUTHENTICATION_METHODS = "OAuth.OpenIDConnect.FAPI." +
             "AllowedClientAuthenticationMethods.AllowedClientAuthenticationMethod";
     public static final String RSA1_5 = "RSA1_5";
+    public static final String PROP_LOGOUT_RETURN_URL = "logoutReturnURL";
+    public static final String LOGOUT_RETURN_URL_DISPLAY_NAME = "Logout Return URL";
```

Comment:
change the dispaly name as URL or Regex

Reply:
In the corresponding SOAP API this value is hardcoded. It was added here as the same for consistency. 

https://github.com/wso2/carbon-identity-framework/pull/3267/files